### PR TITLE
Add tab bar skeleton

### PR DESCRIPTION
**Pull request by JadaTBK**

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 coverage:
   ignore:
-  - Pods/.*
\ No newline at end of file
+  - Pods/.*
+  - Buddies/ViewController.swift
```
Comment (JakeThurman) [inline codecov.yml]: We definitely don't want to do this. I don't think we can just ignore code we don't want to test.

Comment (JadaTBK) [reply codecov.yml]: Yeah, I attempted to do this to test why the coverage went down. I had only made edits to the storyboard but it was saying i had made edits to Buddies/ViewController.swift, if that make sense. I changed this back in the latest commit.

Comment (lukemeierj) [reply codecov.yml]: I think we can (and should?) not worry about code coverage on storyboards.  Apparently codecov is smart enough to do this automatically.  Really weird your codecov dropped like it did.

**Review by lukemeierj — APPROVED** [review]
Looks good to me

**Pull request merged**